src/dds/rs-dds-auto-calibration.cpp

```diff
@@ -0,0 +1,96 @@
+// License: Apache 2.0. See LICENSE file in root directory.
+// Copyright(c) 2024 Intel Corporation. All Rights Reserved.
+
+
+#include "rs-dds-auto-calibration.h"
+#include <rsutils/string/from.h>
+#include <rsutils/json.h>
+#include <src/ds/d500/d500-auto-calibration.h>
+
+
+namespace librealsense
+{
+    dds_auto_calibrated::dds_auto_calibrated() :
```

Comment:
Please do not indent everything in this file.
Use clang-format :)

Reply:
ok